cmd/kubeadm/app/cmd/reset.go

```diff
@@ -83,20 +76,13 @@ func NewReset(skipPreFlight, removeNode bool, certsDir string) (*Reset, error) {
 	}
 
 	return &Reset{
-		removeNode: removeNode,
-		certsDir:   certsDir,
+		certsDir: certsDir,
 	}, nil
 }
 
 // Run reverts any changes made to this host by "kubeadm init" or "kubeadm join".
 func (r *Reset) Run(out io.Writer) error {
 
-	// Try to drain and remove the node from the cluster
-	err := drainAndRemoveNode(r.removeNode)
```

Comment:
I could do it, but I think the node will be marked NotReady by controller manager by default, right?
If we have to set the NodeNotReady taint I'll do it, but I'd like to change the minimum amount of things in this PR